modules/rostests/apitests/shell32/SHChangeNotify.cpp

```diff
@@ -134,13 +114,10 @@ static const TEST_ENTRY s_TestEntries[] = {
     {__LINE__, SHCNE_RMDIR, s_dir2, NULL, "000010000", DoAction2},
     {__LINE__, SHCNE_MKDIR, s_dir2, NULL, "000100000", DoAction1},
     {__LINE__, SHCNE_RENAMEFOLDER, s_dir2, s_dir3, "000000010", NULL},
-    {__LINE__, SHCNE_RENAMEFOLDER, s_dir2, NULL, "000000010", NULL},
```

Comment:
Testing invalid cases is fine. Could that still be done, somehow?

Reply:
I found these tests are invalid in Win10.